package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +315,18 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Getting element information from element column.
+        if all(elements):
+            elements = [i.capitalize() for i in elements]
```

Comment:
As per @lilyminium's comment: https://github.com/MDAnalysis/mdanalysis/pull/2627#discussion_r399722309

Doing a set comparison for elements here would be faster. So:

```python
if all(elements):
    elem_set = set(i.capitalize() for i in i in set(elements))
    if all(element in SYMB2Z for element in elem_set):
        attrs.append(Elements(np.array([i.capitalize() for i in elements], dtype=object))
    else:
        .....
```

Note: I noticed that every other attrs being appended are actually being converted to numpy arrays, so we probably should be consistent here (unless there's a reason not to?).